RedBugs_webapp/src/main/java/com/isa/pl/redbugs/service/PathFindingService.java

```diff
@@ -1,31 +1,48 @@
 package com.isa.pl.redbugs.service;
 
+import com.isa.pl.redbugs.model.Route;
 import com.isa.pl.redbugs.model.Stop;
 import com.isa.pl.redbugs.service.pathfinding.DistanceScorer;
 import com.isa.pl.redbugs.service.pathfinding.Graph;
 import com.isa.pl.redbugs.service.pathfinding.RouteFinder;
 import org.springframework.stereotype.Service;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+
+import java.util.*;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 @Service
 public class PathFindingService {
 
     public Graph<Stop> setUpStopsAndConnections() throws Exception {
```

Comment:
Początek metody mi mówi _biorę **wszystkie** przystanki i wszystkie trasy w bazie danych._  a co jesli będzie w systemie milion przystanków i setki tysięcy tras wywołanie metody spowoduje... łatwo sie domyślić co spowoduje, Czy może lepiej w parametrze przekazać listę przystanków i tras na jakich algortm ma pracować. Znacznie poprawi to performance metody.